extensions/src/test/resources/ActionInvokerListenerTestResources/environment.json

```diff
@@ -0,0 +1,8 @@
+{
+  "environmentVariables": [
```

Comment:
Yes, I agree that would be more simple.  However, I was under the impression that the parameters were going to be passed in an outer environmentVariables object from docker?  If this is not the case I can change it.